Autocomplete select documentation

enisn opened the pull request:
Closes https://github.com/abpframework/abp/issues/11215

enisn committed:
Add initial state of autocomplete select documentation

enisn committed:
Update AutoComplete-Select.md

enisn committed:
Update AutoComplete-Select.md

enisn committed:
Add AutoComplete Select to nav menu of docs

enisn committed:
Revert CmsKit changes

enisn committed:
Spacing fix

enisn committed:
Update AutoComplete-Select.md

hikalkan commented:
@braim23 please check editorial. I will first review, then will inform you.

hikalkan commented on docs/en/UI/AspNetCore/AutoComplete-Select.md:
````diff
@@ -0,0 +1,72 @@
+- `data-autocomplete-selected-item-name`: Text to display as selected item.
+- `data-autocomplete-parent-selector`: jQuery selector expression for parent DOM. _(If it's in a modal, it's suggested to send modal selector as this parameter)_.
+
+    ```html
````
It is better to show the code first (after the "A simple is usage is presented below:" sentence), then explain the options.

hikalkan commented on docs/en/UI/AspNetCore/AutoComplete-Select.md:
````diff
@@ -0,0 +1,72 @@
+    </select>
+    ```
+
+You can define selected option(s) inside select tags. AutoComplete select may can't find selected item if pagination is applied.
````
Can you make this cleaner. I read a few times, but couldn't understand.

hikalkan commented on docs/en/UI/AspNetCore/AutoComplete-Select.md:
````diff
@@ -0,0 +1,72 @@
+    public List<Guid> TagIds { get; set; }
+```
+
+## Possible Issues
````
"Notices" can be a better title. "Issues" makes me feeling that the component has some known bugs.

hikalkan commented on docs/en/UI/AspNetCore/AutoComplete-Select.md:
```diff
@@ -0,0 +1,72 @@
+
+## Possible Issues
+
+### Permission Restriction
```
No need to a second-level title.

hikalkan commented on docs/en/UI/AspNetCore/AutoComplete-Select.md:
```diff
@@ -0,0 +1,72 @@
+### Permission Restriction
+If the authenticated user doesn't have permission on given url, user will get an authorization error. Be careful while designing this kind of UIs.
+
+You may place a lookup method in the same AppService, so your page can retrieve lookup daha of dependent entity without giving a entire read permission to users.
```
Write these two paragraphs as a single paragraph since they are explaining the same thing.
Also, I suggest to write as more generic. Maybe the developer didn't use appservices, but using controllers. Such a suggestion feels that this feature is coupled to appservices. Example: "You can create a specific, unauthorized endpoint/method to get the list of items." (and give a link to the authorization document for the "unauthorized" word).

enisn committed:
Update AutoCompleteSelect documentation according to review

enisn committed:
Update AutoComplete-Select.md

enisn committed:
Update AutoComplete-Select.md

braim23 committed:
Update AutoComplete-Select.md

Please check my changes quickly, if any of them change the meaning you wanted then undo them.

braim23 committed:
Merge branch 'autocomplete-select-documentation' of https://github.com/abpframework/abp into autocomplete-select-documentation

enisn committed:
Update AutoComplete-Select.md
